src/models/searchservice/response/FeaturedSnippetDirectAnswer.ts

```diff
@@ -0,0 +1,22 @@
+import { DirectAnswer } from './DirectAnswer';
+import { DirectAnswerType } from './DirectAnswerType';
+import { Result } from './Result';
+import { Snippet } from './Snippet';
+
+/**
+ * A direct answer which was found within a document.
+ *
+ * @public
+ */
+export interface FeaturedSnippetDirectAnswer extends DirectAnswer {
```

Comment:
do we need/want this extends? It looks like both of the child interfaces have type, value, relatedResult, and verticalKey defined on their own already. Maybe we could flip it on its head and say that a DirectAnswer type is a `FeaturedSnippetDirectAnswer | FieldValueDirectAnswer` and that would let us specify just DirectAnswer in UniversalSearchResponse? (feel like I could be totally off mark here)

Reply:
The extends keyword is typically used so that interfaces can inherit properties from another interface without having to repeat them. That means we could remove the duplicated properties from the other direct answers if we wanted. I put them in there so that they show up in the docs (Nan and I decided on that pattern when making the autocomplete models). Because of that, we could remove the `DirectAnswer` model altogether, but I kind of like having the model to show the common properties)

I agree that it would be nice to have the UniversalSearchResponse return a DirectAnswer, however having the two models explicitly means that typescript forces you to check the type for accessing the properties specific to one of the models, which is really nice. If you don't check the type, Typescript lets you access the common properties. If we have a DirectAnswer which contains all of the possible properties, we would lose that functionality